testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -1067,3 +1245,213 @@ fn retry_on_timeout() {
 
     signer_test.shutdown();
 }
+
+#[test]
+#[ignore]
+/// This test checks that the miner will retry when signature collection times out.
+fn mock_mine_epoch_25() {
```

Comment:
This should be `mock_sign_epoch_25`, right? The rustdoc comment seems like it might need to be updated also.

Reply:
yes it def should be sign.